### PR TITLE
Fix for API change (ITypeMember.ContainingType)

### PR DESCRIPTION
**Pull request by DedSec256**
P.S. 
Also unified the ContainingType declaration through GetContainingType to avoid potential recursive calls in the inheritance hierarchy

### CODE REVIEW

**Comment by DedSec256** [comment]
@ForNeVeR says commented out code needs to be migrated to Azure Toolkit

**Comment by auduchinok** [comment]
Please make the title contain containing types instead of particular API, so it's easier to find related changes and to have an idea of what subsystem was changed.

**Review by ForNeVeR — APPROVED** [review]

File: ReSharper.FSharp/src/FSharp.Psi.Features/src/FileTemplates/AzureFSharpScopeProvider.fs [hunk 1]
```diff
@@ -1,5 +1,5 @@
 namespace JetBrains.ReSharper.Plugins.FSharp.Psi.Features.FileTemplates
-
+(*
```
Comment (auduchinok) [inline ReSharper.FSharp/src/FSharp.Psi.Features/src/FileTemplates/AzureFSharpScopeProvider.fs:2]: What happened to `InAzureFunctionsProject` and the others? cc @ForNeVeR

Comment (auduchinok) [reply ReSharper.FSharp/src/FSharp.Psi.Features/src/FileTemplates/AzureFSharpScopeProvider.fs:2]: @DedSec256 Could you please delete this code completely in a separate PR?

Comment (ForNeVeR) [reply ReSharper.FSharp/src/FSharp.Psi.Features/src/FileTemplates/AzureFSharpScopeProvider.fs:2]: These types were moved to Azure Toolkit plugin; feel free to remove this code from here.

**Review by auduchinok — APPROVED** [review]

**Comment by auduchinok** [comment]
@DedSec256 Thanks!

**Pull request merged**